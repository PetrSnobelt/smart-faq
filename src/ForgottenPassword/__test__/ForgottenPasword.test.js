// @flow

import * as React from 'react';
import { shallow } from 'enzyme';
import ForgottenPasword from '../';

describe('ForgottenPasword', () => {
  const component = shallow(<ForgottenPasword />);
  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
