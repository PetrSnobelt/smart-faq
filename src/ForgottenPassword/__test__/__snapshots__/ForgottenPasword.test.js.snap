// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgottenPasword should match snapshot 1`] = `
<div
  className="jsx-1448086673 ForgottenPassword"
>
  <div
    className="jsx-1448086673 close-icon"
  >
    <Close
      fill="#7f91a8"
      size="32"
    />
  </div>
  <Link
    replace={false}
    to="/kiwi-login"
  >
    <div
      className="jsx-1448086673 back"
    >
      <img
        alt="back"
        className="jsx-1448086673 arrow"
        src="test-file-stub"
      />
      <Typography
        type="secondary"
        variant="bold"
      >
        Back
      </Typography>
    </div>
  </Link>
  <div
    className="jsx-1448086673 picture"
  >
    <img
      alt="Help"
      className="jsx-1448086673"
      src="test-file-stub"
    />
  </div>
  <div
    className="jsx-1448086673 main"
  >
    <p
      className="jsx-1448086673 title"
    >
      Forgotten password
    </p>
    <Typography
      type="secondary"
    >
      Enter your email address and we'll send you the instructions to set your new password.
    </Typography>
    <form
      className="jsx-1448086673"
    >
      <label
        className="jsx-1448086673"
        htmlFor="email"
      >
        Email:
        <input
          className="jsx-1448086673"
          id="email"
          placeholder="e.g. your@email.com"
          type="email"
        />
      </label>
    </form>
    <span
      className="jsx-1448086673 send"
    >
      <Button
        isDisabled={false}
        onClick={[Function]}
        size="large"
        title="Send"
        type="primary"
      />
    </span>
  </div>
  <JSXStyle
    css=".ForgottenPassword.jsx-1448086673{width:480px;padding-top:128px;}div.close-icon.jsx-1448086673{position:absolute;top:12px;right:12px;padding:8px;}div.back.jsx-1448086673{position:absolute;top:18px;left:27px;}img.arrow.jsx-1448086673{width:6px;height:8px;object-fit:contain;margin-right:4px;}div.picture.jsx-1448086673 img.jsx-1448086673{width:203px;height:156px;}div.picture.jsx-1448086673{margin-left:150px;margin-bottom:68px;}div.main.jsx-1448086673{margin-left:40px;margin-right:40px;}p.title.jsx-1448086673{color:#171b1e;font-size:28px;font-weight:bold;margin-bottom:8px;line-height:1.2;}.send.jsx-1448086673{float:right;}form.jsx-1448086673{margin-top:32.8px;}input.jsx-1448086673{width:400px;height:44px;border-radius:3px;background-color:#ffffff;border:solid 1px #bac7d5;margin-bottom:34px;margin-top:4px;font-size:16px;line-height:1.25;padding:12px 24px 12px 16px;}input.jsx-1448086673::-webkit-input-placeholder{width:315px;height:20px;font-size:16px;line-height:1.25;text-align:left;color:#bac7d5;}input.jsx-1448086673::-moz-placeholder{width:315px;height:20px;font-size:16px;line-height:1.25;text-align:left;color:#bac7d5;}input.jsx-1448086673:-ms-input-placeholder{width:315px;height:20px;font-size:16px;line-height:1.25;text-align:left;color:#bac7d5;}input.jsx-1448086673::placeholder{width:315px;height:20px;font-size:16px;line-height:1.25;text-align:left;color:#bac7d5;}label.jsx-1448086673{width:352px;height:20px;font-size:14px;line-height:1.43;text-align:left;color:#46515e;display:inline-table;}"
    styleId="1448086673"
  />
</div>
`;
